### ISSUE TITLE
[Linux Mint 21] Can't build

### ISSUE THREAD

**Issue by SamuelTheTechnoKid** [Rubberduckycooly/Sonic-Mania-Decompilation#104]
When I try building, at the very end, this error appears:
```
done SonicMania/Objects/All.c
linking game...
/usr/bin/ld: /usr/lib/gcc/x86_64-linux-gnu/11/crtbeginT.o: relocation R_X86_64_32 against hidden symbol `__TMC_END__' can not be used when making a shared object
/usr/bin/ld: no se han podido establecer los tamaños de las secciones dinámicas: bad value
collect2: error: ld returned 1 exit status
make: *** [Makefile:128: bin/Linux/SonicMania.so] Error 1
```
Any help, please?

**Comment by LittlePlanetCD** [comment]
have you tried building with STATIC=0 ?

**Comment by MGRich** [comment]
Fixed in 6f23fd83ab3a128d4b609c4116f550497024f996.
